### PR TITLE
fix output of NTT_DATA_DIR and K3_BUILTINS

### PR DESCRIPTION
**Pull request by moosq**

### CODE REVIEW

File: cmd/ntt/show.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ K3_TTCN3_FILES=(
 {{end}})
 
 K3_BUILTINS=(
```
Comment (5nord) [inline cmd/ntt/show.go:112]: Maybe we should remove this EV? Or rename it to K3_INCLUDES?

File: cmd/ntt/show.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ func show(cmd *cobra.Command, args []string) error {
 			"parameters_file",
 			"hooks_file",
 			"lint_file",
-			"datadir",
+			"k3.data_dir",
```
Comment (5nord) [inline cmd/ntt/show.go:49]: I find Datadir is not very useful. Maybe we should remove it?

**Review by 5nord — APPROVED** [review]

**Comment by 5nord** [comment]
The original name of this variable was NTT_DATADIR and not NTT_DATA_DIR. It probably does not matter, though.

File: cmd/ntt/show.go (hunk 2 above)
Comment (moosq) [reply cmd/ntt/show.go]: you are right. I will remove it. Nonetheless sub-fields will still be accessible via commands like `ntt show -- k3.data_dir`

**Pull request merged**